### DISCUSSION TITLE
How to place executable in a bin directory?

### DISCUSSION THREAD

**Discussion by knutson** [marcelotduarte/cx_Freeze#1027] [Q&A]
Currently, when I build using cxfreeze -c main.py, the build directory has the following structure:
>build
>> lib (dependencies)
>> main (executable)

However, to be more consistent with my other software, I would like the structure to be:
>build
>>lib
>>bin
>>>main

However, I cannot figure out how to accomplish this from the documentation. Does anyone else want to do this or see value in it? If I am the only one and for some reason it's better not to place the executable in a dedicated bin directory, please let me know. I'm pretty new to this sort of thing so I'm still trying to learn.

**Comment by marcelotduarte** [comment]
On Linux, this should be possible, as the base Console is using rpath like `-Wl,-rpath,$ORIGIN/lib -Wl,-rpath,$ORIGIN/../lib`
So, you could move the executable to a child's path. It will find for the ./lib in the child path or in the ../lib path.
But you have to move the executable manually (Can be programmatic in the setup).
